README.md

```diff
@@ -113,6 +113,17 @@ Hiera data bindings. A few notable parameters to take note of:
   **Note** Setting this to `latest` is NOT recommended. It will cause the 
   StackStorm packages to be automatically updated without the proper upgrade steps
   being taken (proper steps detailed here: https://docs.stackstorm.com/install/upgrades.html)
+* `st2::python_version` - Version to Python to use. The default is `'system'` and the 
+  system `python` package will be installed, whatever version that is for your OS.
+  To explicitly install Python 3.6 specify `'3.6'`.
+  **Note** If you're running RHEL we automatically install the EPEL repo for you.
+  If you're on Ubuntu 16.04, we install the `'ppa:deadsnakes/ppa'` PPA:
+  https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa
```

Comment:
For all other installation methods we've made this not be a default thing to do. As we want the user to accept the warnings about installing from a PPA, that they have to accept the security risks of doing so.
Is there a way in puppet to do similar.

e.g. by default the user has to interactively answer that they accept the risks, or they provide an explicit variable to say they accept the risks.

This is so everyone realises the risks of continuging to use ST2 with python3.6 from a PPA on U16. 

See https://github.com/StackStorm/st2-packages/blob/72e5d8c6dc23227b2faf568ea50fc2dc30c28225/scripts/st2bootstrap-deb.sh#L128-L144 for example.

Reply:
I added a flag `python_enable_unsafe_repo` that the user will need to explicitly set to `true` in order to enable the deadsnakes PPA